### ISSUE TITLE
show interfaces switchport - access_vlan missing

### ISSUE THREAD

**Issue by urskog84** [CiscoTestAutomation/genieparser#35]
Find a bug in the parser "show interfaces switchport" 

If the vlan name 99 contians a hifen sign (-) i dont get the 
Access Mode VLAN: 99 (Falback-Data)

output from switch
```
Name: Gi1/0/19
Switchport: Enabled
Administrative Mode: static access
Operational Mode: down
Administrative Trunking Encapsulation: dot1q
Negotiation of Trunking: Off
Access Mode VLAN: 99 (Falback-Data)
Trunking Native Mode VLAN: 1 (default)
Administrative Native VLAN tagging: enabled
Voice VLAN: 98 (Fallback-Voice)
Administrative private-vlan host-association: none
Administrative private-vlan mapping: none
Administrative private-vlan trunk native VLAN: none
Administrative private-vlan trunk Native VLAN tagging: enabled
Administrative private-vlan trunk encapsulation: dot1q
Administrative private-vlan trunk normal VLANs: none
Administrative private-vlan trunk associations: none
Administrative private-vlan trunk mappings: none
Operational private-vlan: none
Trunking VLANs Enabled: ALL
Pruning VLANs Enabled: 2-1001
Capture Mode Disabled
Capture VLANs Allowed: ALL

Protected: false
Unknown unicast blocked: disabled
Unknown multicast blocked: disabled
Appliance trust: none

```

No key 'access_vlan'
```
{'switchport_enable': True,
 'switchport_mode': 'static access',
 'operational_mode': 'down',
 'encapsulation': {'administrative_encapsulation': 'dot1q',
  'native_vlan': '1'},
 'negotiation_of_trunk': False,
 'native_vlan_tagging': True,
 'private_vlan': {'native_vlan_tagging': True, 'encapsulation': 'dot1q'},
 'trunk_vlans': 'all',
 'pruning_vlans': '2-1001',
 'capture_mode': False,
 'capture_vlans': 'all',
 'unknown_unicast_blocked': False,
 'unknown_multicast_blocked': False}
```

I renamed the vlan

output from switch
```
Name: Gi1/0/19
Switchport: Enabled
Administrative Mode: static access
Operational Mode: down
Administrative Trunking Encapsulation: dot1q
Negotiation of Trunking: Off
Access Mode VLAN: 99 (Falback)
Trunking Native Mode VLAN: 1 (default)
Administrative Native VLAN tagging: enabled
Voice VLAN: 98 (Fallback-Voice)
Administrative private-vlan host-association: none
Administrative private-vlan mapping: none
Administrative private-vlan trunk native VLAN: none
Administrative private-vlan trunk Native VLAN tagging: enabled
Administrative private-vlan trunk encapsulation: dot1q
Administrative private-vlan trunk normal VLANs: none
Administrative private-vlan trunk associations: none
Administrative private-vlan trunk mappings: none
Operational private-vlan: none
Trunking VLANs Enabled: ALL
Pruning VLANs Enabled: 2-1001
Capture Mode Disabled
Capture VLANs Allowed: ALL

Protected: false
Unknown unicast blocked: disabled
Unknown multicast blocked: disabled
Appliance trust: none

```

i have the key 'access_vlan' :)
```
{'switchport_enable': True,
 'switchport_mode': 'static access',
 'operational_mode': 'down',
 'encapsulation': {'administrative_encapsulation': 'dot1q',
  'native_vlan': '1'},
 'negotiation_of_trunk': False,
 'access_vlan': '99',
 'native_vlan_tagging': True,
 'private_vlan': {'native_vlan_tagging': True, 'encapsulation': 'dot1q'},
 'trunk_vlans': 'all',
 'pruning_vlans': '2-1001',
 'capture_mode': False,
 'capture_vlans': 'all',
 'unknown_unicast_blocked': False,
 'unknown_multicast_blocked': False}
```

**Comment by jeaubin** [comment]
Thanks, 

We will look into it and let you know

**Comment by urskog84** [comment]
believe it is the same problem for 'voice_vlan'

`Voice VLAN: 98 (Fallback-Voice)`

**Comment by cisco-allen** [comment]
Hello urskog84, 
Can ask which OS you are using?
It helps boost debugging, thanks.

**Comment by tahigash** [comment]
Hi @urskog84 

We've released Genie v19.8 with the fix. 
Could you please upgrade by below command and see if it works?

pip install genie --upgrade

**Comment by urskog84** [comment]
Nice it works i Genie v19.8
```
pip3 list | grep genie.libs.parser
genie.libs.parser            19.8
```
```
In [4]: interfaces['GigabitEthernet1/0/17']
Out[4]:
{'switchport_enable': True,
 'switchport_mode': 'static access',
 'operational_mode': 'down',
 'encapsulation': {'administrative_encapsulation': 'dot1q',
  'native_vlan': '1',
  'native_vlan_name': 'default'},
 'negotiation_of_trunk': False,
 'access_vlan': '99',
 'access_vlan_name': 'Falback-Data',
 'native_vlan_tagging': True,
 'voice_vlan': '98',
 'voice_vlan_name': 'Fallback-Voice',
 'private_vlan': {'native_vlan_tagging': True, 'encapsulation': 'dot1q'},
 'trunk_vlans': 'all',
 'pruning_vlans': '2-1001',
 'capture_mode': False,
 'capture_vlans': 'all',
 'protected': False,
 'unknown_unicast_blocked': False,
 'unknown_multicast_blocked': False}

```
